nitro-node/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@janhq/nitro-node",
+  "version": "1.0.0",
+  "description": "This NodeJS library is a wrapper for Nitro, a lightweight (3mb) inference engine written in C++. See nitro.jan.ai",
+  "main": "dist/index.js",
+  "node": "dist/index.cjs.js",
+  "author": "Jan <service@jan.ai>",
+  "license": "AGPL-3.0",
+  "scripts": {
+    "test": "jest --verbose --detectOpenHandles",
+    "pretest": "node download-nitro.js",
+    "build": "tsc --module commonjs && rollup -c rollup.config.ts",
+    "downloadnitro": "node download-nitro.js",
+    "build:publish": "npm pack",
+    "postinstall": "node postinstall.js"
+  },
+  "exports": {
+    ".": "./dist/index.js",
+    "./main": "./dist/index.cjs.js"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-json": "^6.1.0",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-replace": "^5.0.5",
+    "@types/download": "^8.0.5",
+    "@types/jest": "^29.5.11",
+    "@types/node": "^20.11.4",
+    "@types/os-utils": "^0.0.4",
+    "@types/tcp-port-used": "^1.0.4",
+    "jest": "^29.7.0",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.38.5",
+    "rollup-plugin-define": "^1.0.1",
+    "rollup-plugin-sourcemaps": "^0.6.3",
+    "rollup-plugin-typescript2": "^0.36.0",
```

Comment:
I don't see these dependencies are used anywhere in the module now.

Reply:
Noted, will check and remove unused ones.